schutzbot/deploy.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -euxo pipefail
 
+# fallback for gitlab
+GIT_COMMIT="${GIT_COMMIT:-${CI_COMMIT_SHA}}"
```

Comment:
can we just `s/GIT_COMMIT/CI_COMMIT_SHA/` in the whole file instead of defining an extra variable?